app/models/coin.rb

```diff
@@ -276,6 +276,24 @@ def sparkline
     end
   end
 
+  def github_stats(force_refresh: false)
+    return nil unless github_repo.present?
+
+    Rails.cache.fetch("coins/#{id}/github_stats", force: force_refresh) do
+      response = CoinServices::RetrieveGithubStats.call(coin: self)
+      response.try(:result)
+    end
+  end
+
+  def merge_github_stats(stats_to_merge)
+    return nil unless github_repo.present?
```

Comment:
I think you can just do `return` instead of `return nil` to accomplish the same thing.